doc/changelog.rst

```diff
@@ -1,6 +1,108 @@
 Changelog
 =========
 
+.. _changelog-v2-2-0:
+
+v2.2.0
+------
+
+This release includes
+
+Bug fixes
+~~~~~~~~~
+
+-  Fix path settings for DKRZ/Mistral (`#852 <https://github.com/ESMValGroup/ESMValCore/pull/852>`__) `Bouwe Andela <https://github.com/bouweandela>`__
+-  Change logic for calling the diagnostic script to avoid problems with scripts where the executable bit is accidentally set (`#877 <https://github.com/ESMValGroup/ESMValCore/pull/877>`__) `Bouwe Andela <https://github.com/bouweandela>`__
+-  Fix overwriting in generic level check (`#886 <https://github.com/ESMValGroup/ESMValCore/pull/886>`__) `sloosvel <https://github.com/sloosvel>`__
+-  Add double quotes to script args in rerun screen message when using vprof profiling (`#897 <https://github.com/ESMValGroup/ESMValCore/pull/897>`__) `Valeriu Predoi <https://github.com/valeriupredoi>`__
+-  Simplify time handling in multi-model statistics preprocessor (`#685 <https://github.com/ESMValGroup/ESMValCore/pull/685>`__) `Peter Kalverla <https://github.com/Peter9192>`__
+-  Fix duplication in preprocessors DEFAULT_ORDER  (`#973 <https://github.com/ESMValGroup/ESMValCore/pull/973>`__) `Javier Vegas-Regidor <https://github.com/jvegasbsc>`__
```

Comment:
While this was indeed a bug that was fixed, it was not present in the previous release because the bug was introduced #696 which was merged after that. Therefore I think it doesn't make much sense to list this under bug fixes, it makes more sense to add it to the description of #696.